src/nbl/video/device_capabilities/device_limits.json

```diff
@@ -1760,7 +1760,6 @@
                     "type": "uint32_t",
                     "name": "shaderGroupHandleSize",
                     "value": 32,
-                    "expose": "DISABLE",
```

Comment:
do not expose, its a constexpr essentially, you can add it as a `constexpr static inline` here: https://github.com/Devsh-Graphics-Programming/Nabla/blob/f4b26265f6459bd82f272d1b7060c87f06c5e460/include/nbl/video/SPhysicalDeviceLimits.h#L27